src/main/java/org/isf/medicalstockward/manager/MovWardBrowserManager.java

```diff
@@ -224,7 +224,40 @@ public List<MedicalWardForPrint> convertWardDrugs(Ward wardSelected, List<Medica
 		Collections.sort(drugPrint);
 		return drugPrint;
 	}
-
+	
+	/**
+	 * Get the  MedicalWard for the specified criteria.
+	 *
+	 * @param wardCode - the ward id  
+	 * @param medical - the mecical id 
+	 * @param lot - the lot id
```

Comment:
```suggestion
	 * @param lotCode - the lot id
```
Use the correct parameter name.